browser/playlist/test/BUILD.gn

```diff
@@ -3,7 +3,7 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import("//brave/components/playlist/buildflags/buildflags.gni")
+import("//brave/components/playlist/common/buildflags/buildflags.gni")
 
 source_set("browser_tests") {
   assert(enable_playlist)
```

Comment:
pls move this to the top level like so:
```
import("//brave/components/playlist/common/buildflags/buildflags.gni")

assert(enable_playlist)

source_set("browser_tests") {
```

and remove it from `source_set("unit_tests")`